sophys_gui/functions.py

```diff
@@ -101,7 +101,8 @@ def handleSpinboxWidget(valueType):
     else:
         spinbox = QSpinBox()
     spinbox.setMaximumHeight(50)
-    spinbox.setMaximum(10000)
+    spinbox.setMinimum(-10**5)
+    spinbox.setMaximum(10**5)
```

Comment:
I know this was changed because of the issue, but that should be documented either in the commit message, or the in pull request (if you were to open another one, which would be ideal).